server/services/ImageUpload.js

```diff
@@ -0,0 +1,36 @@
+const aws = require("aws-sdk");
+const multer = require("multer");
+const multerS3 = require("multer-s3");
+
+const s3 = new aws.S3();
+
+aws.config.update({
+  secretAccessKey: process.env.S3_SECRET,
+  accessKeyId: process.env.S3_ACCESS_KEY,
+  region: "ca-central-1",
+});
+
+const fileFilter = (req, file, cb) => {
+  if (file.mimetype === "image/jpeg" || file.mimetype === "image/png") {
```

Comment:
👍🏽

Reply:
Thank you. I will allow jpeg and png in FE too.